### PR TITLE
IoT Trigger Templates

### PR DESCRIPTION
**Pull request by ElleTojaroon**
after rebase

### CODE REVIEW

File: Functions.Templates/Bindings/bindings.json [hunk 1]
```diff
@@ -1910,8 +1914,12 @@
           "name": "disable",
           "type": "NAND",
           "values": [
-            { "value": "idToken" },
```
Comment (alrod) [inline Functions.Templates/Bindings/bindings.json]: Please move back O365 stuff

File: Functions.Templates/Bindings/bindings.json [hunk 2]
```diff
@@ -2278,590 +2286,6 @@
           "required": true,
           "label": "$oneDrive_out_name_label",
           "help": "$oneDrive_out_name_help"
-        },
```
Comment (alrod) [inline Functions.Templates/Bindings/bindings.json]: Please move back O365 stuff

File: Functions.Templates/Bindings/bindings.json [hunk 3]
```diff
@@ -1910,8 +1914,12 @@
           "name": "disable",
           "type": "NAND",
           "values": [
-            { "value": "idToken" },
-            { "value": "PrincipalId" }
```
Comment (alrod) [inline Functions.Templates/Bindings/bindings.json]: Please move back O365 stuff

File: Functions.Templates/Templates/IoTHubTrigger-TypeScript/index.js.map [hunk 4]
```diff
@@ -0,0 +1 @@
+{"version":3,"file":"index.js","sourceRoot":"","sources":["index.ts"],"names":[],"mappings":";;AAAA,aAAoB,OAAY,EAAE,cAAqB;IACnD,OAAO,CAAC,GAAG,CAAC,mEAAiE,cAAgB,CAAC,CAAC;IAE/F,cAAc,CAAC,OAAO,CAAC,UAAA,OAAO;QAC1B,OAAO,CAAC,GAAG,CAAC,uBAAqB,OAAS,CAAC,CAAC;IAChD,CAAC,CAAC,CAAC;IAEH,OAAO,CAAC,IAAI,EAAE,CAAC;AACnB,CAAC;AARD,kBAQC;AAAA,CAAC"}
```
Comment (alrod) [inline Functions.Templates/Templates/IoTHubTrigger-TypeScript/index.js.map]: I think we do not need this file

File: Functions.Templates/Templates/IoTHubServiceBusTopicTrigger-CSharp/metadata.json [hunk 5]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "IoTHubTrigger (ServiceBusTopic)",
+  "defaultFunctionName": "IoTHubServiceBusTopicTriggerCSharp",
+  "description": "A C# function that will be run whenever a message is added to the specified Service Bus topic",
```
Comment (alrod) [inline Functions.Templates/Templates/IoTHubServiceBusTopicTrigger-CSharp/metadata.json]: move to resources

File: Functions.Templates/Bindings/bindings.json (hunk 1 above)
Comment (ElleTojaroon) [inline Functions.Templates/Bindings/bindings.json]: I checked. This is the same as the code in azure git hub repo

---
In reply to: [133859502](https://github.com/Azure/azure-webjobs-sdk-templates/pull/539#discussion_r133859502) [](ancestors = 133859502)

File: Functions.Templates/Bindings/bindings.json (hunk 3 above)
Comment (ElleTojaroon) [inline Functions.Templates/Bindings/bindings.json]: I checked. This is the same as the code in azure git hub repo

---
In reply to: [133859524](https://github.com/Azure/azure-webjobs-sdk-templates/pull/539#discussion_r133859524) [](ancestors = 133859524)

File: Functions.Templates/Bindings/bindings.json (hunk 2 above)
Comment (ElleTojaroon) [inline Functions.Templates/Bindings/bindings.json]: added

---
In reply to: [133859511](https://github.com/Azure/azure-webjobs-sdk-templates/pull/539#discussion_r133859511) [](ancestors = 133859511)

File: Functions.Templates/Templates/IoTHubTrigger-TypeScript/index.js.map (hunk 4 above)
Comment (ElleTojaroon) [inline Functions.Templates/Templates/IoTHubTrigger-TypeScript/index.js.map]: deleted .js and .js.map


---
In reply to: [133859668](https://github.com/Azure/azure-webjobs-sdk-templates/pull/539#discussion_r133859668) [](ancestors = 133859668)

**Comment by soninaren** [comment]
added this commit https://github.com/Azure/azure-webjobs-sdk-templates/commit/df94e19484fea88fc2c68d9f032c9d18d860d5b5

**Pull request closed without merging**